tensorflow/contrib/learn/python/learn/monitors.py

```diff
@@ -42,9 +42,6 @@
 import numpy as np
 import six
 
-from tensorflow.contrib.framework import deprecated
-from tensorflow.contrib.framework.python.ops import variables as contrib_variables
-from tensorflow.contrib.learn.python.learn.summary_writer_cache import SummaryWriterCache
```

Comment:
All those are not needed? Good find!

Reply:
Ah I actually update this code some time ago, and realized that these import statements were no longer needed.